image/templates/helm/stackrox-secured-cluster/templates/sensor-scc.yaml

```diff
@@ -1,47 +1,59 @@
 {{- include "srox.init" . -}}
 
-{{- if and ._rox.env.openshift ._rox.system.createSCCs }}
-
-apiVersion: security.openshift.io/v1
-kind: SecurityContextConstraints
+{{- if ._rox.env.openshift }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
 metadata:
-  name: stackrox-sensor
+  name: sensor-scc
+  namespace: {{ .Release.Namespace }}
   labels:
-    {{- include "srox.labels" (list . "securitycontextconstraints" "stackrox-sensor") | nindent 4 }}
-    auto-upgrade.stackrox.io/component: "sensor"
+    {{- include "srox.labels" (list . "role" "sensor-scc") | nindent 4 }}
   annotations:
-    {{- include "srox.annotations" (list . "securitycontextconstraints" "stackrox-sensor") | nindent 4 }}
-    kubernetes.io/description: stackrox-sensor is the security constraint for the sensor
-users:
-  - system:serviceaccount:{{ ._rox._namespace }}:sensor
-  - system:serviceaccount:{{ ._rox._namespace }}:sensor-upgrader
-priority: 0
-runAsUser:
-  type: RunAsAny
-seLinuxContext:
-  type: RunAsAny
-seccompProfiles:
-  - '*'
-supplementalGroups:
-  type: RunAsAny
-fsGroup:
-  type: RunAsAny
-groups: []
-readOnlyRootFilesystem: true
-allowHostDirVolumePlugin: false
-allowHostIPC: false
-allowHostNetwork: false
-allowHostPID: false
-allowHostPorts: false
-allowPrivilegeEscalation: true
-allowPrivilegedContainer: false
-allowedCapabilities: []
-defaultAddCapabilities: []
-requiredDropCapabilities: []
-volumes:
-  - configMap
-  - downwardAPI
-  - emptyDir
-  - secret
-
+    {{- include "srox.annotations" (list . "role" "sensor-scc") | nindent 4 }}
+rules:
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  resourceNames:
+  - restricted-v2
```

Comment:
No need for a Role for `restricted-v2`

Reply:
Removed in https://github.com/stackrox/stackrox/pull/8663/commits/4b664b9d6c8213acfac453f44bf713eda1ffe002